skglm/solvers/cd_solver.py

```diff
@@ -226,32 +221,38 @@ def cd_solver(
     obj_out = []
     all_feats = np.arange(n_features)
     stop_crit = np.inf  # initialize for case n_iter=0
-    w_acc, Xw_acc = np.zeros(n_features), np.zeros(n_samples)
+    w_acc, Xw_acc = np.zeros(n_features+fit_intercept), np.zeros(n_samples)
```

Comment:
use spaces around + 

they are not mandatory, but we use them everywhere else. Staying consistent allows more efficient `ctrl F` + replace.